pages/addCommunity.tsx

```diff
@@ -0,0 +1,96 @@
+import { useState } from 'react';
+
+import Button from '../components/button/Button';
+import Card from '../components/card/Card';
+import request from '../modules/request';
+import { Form, Input, Label } from '../styles/AddCommunity.styles';
+import { AppWrapper, GlobalStyle, Navigation } from '../styles/index.styles';
+
+const AddCommunity = (): JSX.Element => {
+  const [title, setTitle] = useState('');
+  const [url, setUrl] = useState('');
+  const [members, setMembers] = useState('');
+  const [image, setImage] = useState('');
+  const isEnabled =
+    title.length > 0 &&
+    url.length > 0 &&
+    members.length > 0 &&
+    image.length > 0;
+
+  console.log('title:', title, 'Url:', url, 'mem:', members, 'image', image);
+
+  const submitForm = async () => {
+    const query = `
+      mutation {
+        add(title: "${title}", url: "${url}", members: ${members}, image: "${image}"){
+          title
+          url
+          members
+          image
+        }
+      }
+    `;
+
+    const response = await request(query);
+
+    //Function run after 10sec
+    setTimeout(() => {
+      alert(`Success!The ${title} community added!`);
+      //clear form
+      Array.from(document.querySelectorAll('input')).forEach(
+        (input) => (input.value = '')
+      );
+    }, 10000);
+  };
+
+  return (
+    <AppWrapper>
+      <GlobalStyle />
+      <h1>WELCOME</h1>
```

Comment:
Welcome to "Add community"? :stuck_out_tongue_closed_eyes:

Reply:
Done!